--- a/ssb-emulator/Cargo.lock
+++ b/ssb-emulator/Cargo.lock
@@ -641,7 +641,6 @@
 [[package]]
 name = "skytemple_rust"
 version = "1.6.0"
-source = "git+https://github.com/SkyTemple/skytemple-rust.git#48a7a5f58d3416507bb28d1b123ab07a3bc05fb3"
 dependencies = [
  "bytes",
  "encoding",

--- a/ssb-emulator/Cargo.toml
+++ b/ssb-emulator/Cargo.toml
@@ -12,7 +12,7 @@
 
 [dependencies]
 desmume-rs = "0.1"
-skytemple_rust = { git = "https://github.com/SkyTemple/skytemple-rust.git", default-features = false, features = ["script_var_table"] }
+skytemple_rust = { path = "../skytemple-rust", default-features = false, features = ["script_var_table"] }
 pyo3 = { version = "0.18", features = ["extension-module"] }
 pyo3-log = "0.8"
 log = "0.4"
