--- a/ssb-emulator/setup.py
+++ b/ssb-emulator/setup.py
@@ -13,7 +13,7 @@
 setup(
     name="skytemple-ssb-emulator",
     version=__version__,
-    rust_extensions=[RustExtension(f"skytemple_ssb_emulator", binding=Binding.PyO3)], # set debug=True for unoptimized build.
+    rust_extensions=[RustExtension(f"skytemple_ssb_emulator", binding=Binding.PyO3, cargo_manifest_args=["--offline"])], # set debug=True for unoptimized build.
     packages=["skytemple_ssb_emulator"],
     package_data={"skytemple_ssb_emulator": ["py.typed", "*.pyi"]},
     install_requires=[
